snuba/datasets/schemas/__init__.py

```diff
@@ -1,12 +1,18 @@
 from __future__ import annotations
 
 from abc import ABC, abstractmethod
-from typing import Mapping, List, Sequence
+from typing import List, Mapping, NamedTuple, Sequence
 
 from snuba.clickhouse.columns import ColumnSet, ColumnType
+from snuba.query.expressions import Expression
 from snuba.query.types import Condition
 
 
+class MandatoryCondition(NamedTuple):
+    legacy: Condition
+    ast: Expression
```

Comment:
Should it be `FunctionCall`? IIRC condition is always a function in the AST.

Reply:
Indeed, I think I can make this a FunctionCall.